src/tool_help.c

```diff
@@ -928,6 +928,7 @@ void tool_help(const char *category)
     puts("Invalid category provided, here is a list of all categories:\n");
     get_categories();
   }
+  free((char *)category);
```

Comment:
Why cast the item?

Reply:
because without it, there's a compiler warning. It's a `const char *`

Reply:
I removed the `const` in a follow-up commit and then I can remove the typecast.

Reply:
If it can be modified (in this case freed) it should probably not be const in the first place...

Reply:
@jzakrzewski exactly what I arrived at after some more thinking! =)

Reply:
@bagder You're simply too fast for me too keep up ;)